tests/core/test_actions.py

```diff
@@ -510,6 +510,21 @@ async def test_action_session_start_without_slots(
     assert events == [SessionStarted(), ActionExecuted(ACTION_LISTEN_NAME)]
 
 
+async def test_action_session_start_with_metada(
```

Comment:
```suggestion
async def test_action_session_start_with_metadata(
```

Reply:
Also, I think a unit test for this change should be more "high level", since we've modified the code all the way from `log_message()`